fontforge/parsepdf.c

```diff
@@ -1565,6 +1562,8 @@ static void _InterpretPdf(FILE *in, struct pdfcontext *pc, EntityChar *ec) {
 		    tok==pt_fillstrokeeo || tok==pt_closefillstrokenz ||
 		    tok==pt_closefillstrokeeo )
 		ent->u.splines.fill.col = fore_fill;
+		// FIXME Where to set?
+		// memcpy(ent->u.splines.stroke.dashes,dashes,sizeof(dashes));
```

Comment:
Note: the bot above (@lgtm-com) is registering a new FIXME…I don't know how to use its website and don't care to learn so I checked out 2f86e366a58b97a36873253a1bc59869acebb209 of jtanx/fontforge cf2 branch and then compared it to HEAD. It's right here. I thought it was probably a bug but it's not, @jtanx really added a FIXME. Probably does not matter, would be good to fix in ideal world, which we don't live in. But, in case anyone else wants to know what bot is complaining about, this is it.